ga4gh/datamodel/reads.py

```diff
@@ -76,3 +158,66 @@ def toProtocolElement(self):
         readGroup.referenceSetId = None
         readGroup.sampleId = None
         return readGroup
+
+    def getReadAlignments(self, referenceName=None, referenceId=None,
```

Comment:
See https://github.com/ga4gh/schemas/pull/193 for a revision to this API that strips the referenceName argument. While both `referenceName` and `referenceId` continue to be supported in the v0.5.1 version of the API (thus I'm not suggesting we change this here), we should also not spend too much time supporting the case where both arguments are supplied, as it's going to go away soon.